doc/StereoStreaming.md

```diff
@@ -31,7 +42,7 @@ payload. This message is silently ignored by older Servers.
 
 ## Examples
 
-Example of a stereo 3D client application using the blocking Stream API:
+Example of a stereo 3D client application using synchronous operations:
```

Comment:
must they? If executed as is, each call should be synchronous, or am I missing something?